#pragma once
#include <exception>

void handleException(const std::exception &e);
